### ISSUE TITLE
Is it possible to override :jvm-opts in a project's shadow-cljs.edn?

### ISSUE THREAD

**Issue by socksy** [thheller/shadow-cljs#794]
Hi there! I work on a project where the maintainers have overridden the default maximum memory in the jvm opts, and on my machine that often means I have to kill the shadow process when I want to do other memory intensive tasks, such as looking at a slack thread, or starting a zoom call. 

Ideally, I would be able to override this value on my local machine. I've tried adding `jvm-opts ["-Xms1G" "-Xmx2G"]` to my `~/.shadow-cljs/config.edn`, but I believe this is not being respected. I can't find any documentation on this, but I was wondering if it would be possible to pass an env var in somehow?

**Comment by thheller** [comment]
Is this with just `shadow-cljs.edn` or using `deps.edn` or `project.clj`?

**Comment by socksy** [comment]
We are using `deps.edn`, but no JVM options are set there, only in `shadow-cljs.edn`. Actually, does that mean that all of the `:jvm-opts` are being ignored?

**Comment by thheller** [comment]
You can just skip using the `shadow-cljs` command and just use `clj` directly. That lets you set whatever you want already.

```
clj -J-Xmx=1G -A:your-alias shadow.cljs.devtools.cli watch app
# would be basically but with extra JVM opts
shadow-cljs watch app
```

**Comment by socksy** [comment]
Thanks for the tip — it's not really practical for me to run `clj` by itself for this project because the command line arguments are all being dynamically generated by a python script we use for our dev tooling. 

But your tip sent me down a path that I managed to solve it (I hope) and I'll document it here on the off-chance it helps others:

1. I moved the `jvm-opts ["-Xms2G" "-Xmx4G"]` line that was in the `shadow-cljs.edn` into an alias `:normal-mem in the `deps.edn`
2. I added another alias, `:low-mem`, which caps at 1G, giving me the following two lines in the aliases:
```clojure
:normal-mem {:jvm-opts ["-Xms1G" "-Xmx4G"]}
:low-mem {:jvm-opts ["-Xmx1G"]}
```
3. I adapted the python script so that if `--low-mem` was passed to it, it would add the `low-mem` alias, and otherwise add the `normal-mem` to the alias list

I tested and it works, so it's now just a matter of persuading other people to get the code in :) Thanks for your help!
